src/Components/Categories/CategorieServices.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios'
```

Comment:
this file is not scope of this user story, please delete it from this PR

Reply:
Oops. Sorry. Done!